modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,30 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section explains the similarities and differences in these Mule versions.
+
+Policies in Mule 4
+Classloader isolation exists between the application, the runtime and connectors, and policies. To fulfill the isolation requirement, policies are self-contained and packaged to include libraries needed for execution, similar to an application. Additionally, certain changes to the packaging, size, and scope of a policy are introdcued in Mule 4.
+
+[%header,cols="40a,50a,50a"]
+|===
+| Component | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
+| Policies distribution | Policies cannot be distributed outside of runtime | You can distribute policies outside of the runtime, which simplifies upgrades 
```

Comment:
What does distributed means in this case? seems confusing. @balbifm do you remember what this meant?

Reply:
Not really, where does this come from?

Reply:
I can't recall why I added that, but I guess what it means is that you can't implement policies outside of runtime. Does that make sense? Or should I remove it?